packages/common/core-utils/src/timer.ts

```diff
@@ -260,7 +260,7 @@ export interface IPromiseTimer extends ITimer {
  * @internal
  */
 export class PromiseTimer implements IPromiseTimer {
-	private deferred?: Deferred<IPromiseTimerResult>;
+	private deferred?: Deferred<IPromiseTimerResult> | undefined;
```

Comment:
My preference is typically to omit the `?` for class members, and is the more common pattern in the repo.

```suggestion
	private deferred: Deferred<IPromiseTimerResult> | undefined;
```